### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Dershowitz011**
Directly fetch the latest version by Carthage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,7 +181,7 @@ end
 
 Then to use the component in your code, add the following line to your `.h` or `.m` files:
 
-```
+```swift
```
Comment (gergelyorosz) [inline README.md]: thanks for noticing the missing language specifier. In this case, this code is Objective C. Could you please change this to `objc` over `swift`?

**Comment by gergelyorosz** [comment]
@Dershowitz011 thanks for the PR! Just one tiny comment on specifying the language for Objective C, and then good to merge.

**Comment by Dershowitz011** [comment]
Done.

**Comment by gergelyorosz** [comment]
:boom: thanks very much!

**Pull request merged**